index.html

```diff
@@ -65,54 +77,70 @@ <h4>Skills</h4>
     </section>
     <section id="recent-works">
         <h2>MY RECENT WORKS</h2>
-        <hr>        <div id="recent-works-grid">
+        <hr>        
+        <div id="recent-works-grid">
```

Comment:
Figma Design
![Screenshot from 2022-04-30 21-10-23](https://user-images.githubusercontent.com/77842237/166117396-afc1cc9b-5d96-4128-95d9-18018c90f677.png)

Your project 
![Screenshot from 2022-04-30 21-11-14](https://user-images.githubusercontent.com/77842237/166117410-ffb8dbca-7951-4ce4-9aea-e3bc644ea420.png)


- [ ] Kindly there should be padding on top between the line of the recent project and the first project.